### DISCUSSION TITLE
Wireless Drivers  Not found

### DISCUSSION THREAD

**Discussion by NathanRichardson8** [elementary/triage#495] [Q&A]
I have installed Elementary however the wireless is not working. lshw -C Network says Configuration: driver=bcma-pci-bridge. A "sudo lsmod" confirms i can see bcma, is this the right driver? But when I do a "sudo modprobe bcma-pci-bridge" to load the driver it cannot be found.

Not sure what to do next. I did download the b43 driver. thanks

**Comment by NathanRichardson8** [comment]
Another observation, when I tried to install the driver "sudo apt install firmware-b43-installer" it gave an error message "Failure resolving archive.ubuntu.com" and then unable to fetch archives.

**Reply by teamcons** [reply]
im not sure how you would expect it to fetch archives without internet connection. You can download it manually from another device, and install the file with dpkg from an usb stick

**Comment by alvaromellado** [comment]
Can you mention what card do you have? (For example: Intel AX200 or MT9721)

**Comment by nicekiwi** [comment]
Not sure if related or not, but I had issues installing the `[bcmwl-kernel-source](https://launchpad.net/ubuntu/+source/bcmwl)`  driver because gcc-12 was not installed. If you install `gcc-12` this driver should install. 

I had to use USB hotspot tethering from my Phone for connectivity to do this.
